browser/ChangeLog.md

```diff
@@ -1,5 +1,17 @@
 Note: This is the change log file for Azure Storage JavaScript Client Library.
 
+2018.06 Version 2.10.100
```

Comment:
what's the min version 100 stands for?

Reply:
To differ from the Node.js version number. Every JS new minor version starts with patch version 100.